presto-docs/src/main/sphinx/installation/cli.rst

```diff
@@ -45,3 +45,40 @@ which is configured with a carefully selected set of options. This behavior
 can be overridden by setting the environment variable ``PRESTO_PAGER`` to the
 name of a different program such as ``more`` or `pspg <https://github.com/okbob/pspg>`_,
 or it can be set to an empty value to completely disable pagination.
+
+History
+-------
+
+The CLI keeps a history of your previously used commands. You can access your
+history by scrolling or searching. Use the up and down arrows to scroll and
+:kbd:`Control+S` and :kbd:`Control+R` to search. To execute a query again,
+press **Enter**.
+
+By default, you can locate the Presto history file in ``~/.presto_history``.
+Use the ``PRESTO_HISTORY_FILE`` environment variable to change the default.
+
+Troubleshooting
+---------------
+
+If something goes wrong, an error message displays:
+
+.. code-block:: none
+
+    $ presto
+    presto> select count(*) from tpch.tiny.nations;
+    Query 20200804_201646_00003_f5f6c failed: line 1:22: Table 'tpch.tiny.nations' does not exist
+    select count(*) from tpch.tiny.nations
+
+To view debug information when using the CLI:
```

Comment:
How about

```
To view debug information, including the stack trace for failures, use the ``--debug`` option:
```
I want to mention "stack trace" since that's the most common thing needed for debugging a failed query.

Reply:
Thats a good suggestions